### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by liujianzhao6328057** [foamliu/InsightFace-v3#1]
Hi,foamLiu. Thanks for your pytorch implementation of arcface ! Could you please share the weight file？Thank you very much.

**Comment by foamliu** [comment]
Here you are: https://github.com/foamliu/InsightFace-v3/releases/download/v1.0/insight-face-v3.pt.

**Comment by xsacha** [comment]
Nothing newer than May?
It seems there has been many changes since but it is hard to follow without commit messages.

I can't reproduce your megaface score (not even close) on prebuilt so I was assuming it is just old. The README from May shows the score a full 1% lower. It also seems the traced version may be bugged.

**Comment by foamliu** [comment]
The model is right, may I ask how did you align FaceScrub?
